<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>339</width>
      <height>641</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>旋转和平移</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>预设视角</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="2">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>item</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Row</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Item And Row</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Column</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Item And Column</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Row And Column</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Item Row And Column</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Row Slice</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Column Slice</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>水平旋转</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QSlider" name="horizontalSlider">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0" rowspan="2" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>垂直旋转</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>缩放</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QSlider" name="horizontalSlider_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="4" rowspan="4">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::GoUp"/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::EditUndo"/>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::GoNext"/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::GoDown"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QSlider" name="horizontalSlider_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>图形总体</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>66</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>图形主题</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>66</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>字体大小</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>66</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>选择模式</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>30</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>60</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>90</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>92</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>显示背景</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>92</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>显示倒影</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>92</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>显示轴标题</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>120</y>
          <width>92</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>显示背景网格</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_5">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>150</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>数值坐标轴反向</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_6">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>180</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>显示轴标签背景</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>序列设置</string>
       </property>
       <widget class="QCheckBox" name="checkBox_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>92</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>光滑效果</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_8">
        <property name="geometry">
         <rect>
          <x>101</x>
          <y>120</y>
          <width>161</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>显示选中棒柱的标签</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>66</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>棒柱样式</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_4">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>86</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="action_3"/>
   <addaction name="separator"/>
   <addaction name="action_4"/>
   <addaction name="action_5"/>
   <addaction name="action_6"/>
  </widget>
  <action name="action">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::InsertImage"/>
   </property>
   <property name="text">
    <string>序列颜色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditRedo"/>
   </property>
   <property name="text">
    <string>重新生成</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCut"/>
   </property>
   <property name="text">
    <string>修改数值</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MailForward"/>
   </property>
   <property name="text">
    <string>添加行</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
   </property>
   <property name="text">
    <string>插入行</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditClear"/>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
